UnitySDK/Assets/ML-Agents/Editor/AgentEditor.cs

```diff
@@ -24,7 +23,7 @@ public override void OnInspectorGUI()
                 "agentParameters.maxStep");
             SerializedProperty isResetOnDone = serializedAgent.FindProperty(
                 "agentParameters.resetOnDone");
-            SerializedProperty isODD = serializedAgent.FindProperty(
+            SerializedProperty isOdd = serializedAgent.FindProperty(
```

Comment:
Should we renamed this to e.g. `isOnDemandDecision` instead? Or is that a pain because it's serialized?

Reply:
we can annotate it with [FormerlySerializedAsAttribute](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html)

Reply:
Upon closer inspection, it seems like this is doing a string key search for a local variable, so it should be fine renaming it without having to worry about how it's serialized.  If we changed the member names on agentParameters, we'd need changes here as well.